### ISSUE TITLE
no message when print done

### ISSUE THREAD

**Issue by KrisRux** [OctoPrint/OctoPrint-Pushbullet#26]
i don't receive the messago when print done but test and update function work correctly.

**Comment by mazamorac** [comment]
I've never received finished print job notifications, and progress notifications started failing every once in a while. 
Looking in the octoprint log files, I see the startup message, and when it fails:
  ```
2020-04-22 16:09:32,768 - octoprint.plugin - ERROR - Error while calling plugin octobullet
Traceback (most recent call last):
  File "/home/pi/git/OctoPrint/venv/local/lib/python2.7/site-packages/octoprint/plugin/__init__.py", line 224, in call_plugin
    result = getattr(plugin, method)(*args, **kwargs)
  File "/home/pi/git/OctoPrint/venv/local/lib/python2.7/site-packages/octoprint_octobullet/__init__.py", line 241, in on_event
    path = os.path.basename(payload["file"])
KeyError: 'file'
```

My guess is that it's looking for the name of the file via a key "file" in a duct, and bombs (I use "file" in my status message config).

The call to getattr() should probably be wrapped in an exception handler. If I get a chance I might submit a fix.

The workaround would be to omit the {file} in the status message, at least. There may be others that error out too.

**Comment by scottrini** [comment]
Hey there!  I actually just recently took over maintaining this plugin because I ran into this specific issue.  The key in octoprint changed, so instead of `file`, it's now called `name`.  I have a fork with this update applied: https://github.com/scottrini/OctoPrint-Pushbullet

I'm going to get this merged into this repo and get it tagged and bumped.

**Comment by scottrini** [comment]
Fixed in [0.1.10](https://github.com/OctoPrint/OctoPrint-Pushbullet/releases/tag/0.1.10)
